examples/animation/animated_fox.rs

```diff
@@ -4,6 +4,7 @@ use std::f32::consts::PI;
 use std::time::Duration;
 
 use bevy::{animation::RepeatAnimation, pbr::CascadeShadowConfigBuilder, prelude::*};
+use bevy_internal::gltf::GltfLoaderSettings;
```

Comment:
no bevy_internal in examples even if your IDE really really want to import it :D